doc/m0sched.rst

```diff
@@ -0,0 +1,44 @@
+This document contains information about m0scheduler python3 script. 
+
+**m0scheduler** responsible for,
+
+- getting **m0sched** parameters from Hare
```

Comment:
I think, now, its name is `fdmi_sample_plugin`

Reply:
This file is obsolete and the source code in the comments contains much fresh and useful info. I removed the file in the last commit.